awsenergylabelerlib/awsenergylabelerlib.py

```diff
@@ -34,10 +34,12 @@
 import logging
 import re
 from collections import Counter
+import requests
 
 import pandas as pd
 
-from .awsenergylabelerlibexceptions import InvalidAccountListProvided, MutuallyExclusiveArguments
+from .awsenergylabelerlibexceptions import InvalidAccountListProvided, InvalidRegionListProvided, \
```

Comment:
Since the imports are longer than a line prefer the import style with parentheses and imports aligned verticaly.